narwhals/dataframe.py

```diff
@@ -443,23 +443,27 @@ def write_csv(self, file: str | Path | BytesIO | None = None) -> Any:
 
             >>> import pandas as pd
             >>> import polars as pl
+            >>> import pyarrow as pa
             >>> import narwhals as nw
             >>> df = {"foo": [1, 2, 3], "bar": [6.0, 7.0, 8.0], "ham": ["a", "b", "c"]}
             >>> df_pd = pd.DataFrame(df)
             >>> df_pl = pl.DataFrame(df)
+            >>> df_pa = pa.DataFrame(df)
```

Comment:
Hey, if you want to create a pyarrow table you should do it with pa.table(data).
```suggestion
            >>> df_pa = pa.table(df)
```